Makefile

```diff
@@ -21,7 +21,7 @@ fmt :
 	$(GO_FMT) -s -w ./
 
 test :
-	go test -race -coverprofile=coverage.txt -covermode=atomic ./router ./msgservice ./dbservice ./formatting ./data ./regoservice ./routes
+	go test -race -v ./...
```

Comment:
we could add this params too:  -v -short -race -timeout 30s -coverprofile=coverage.txt ./...